sites/partners/src/listings/helpers.ts

```diff
@@ -0,0 +1,12 @@
+import { FormListing } from "./PaperListingForm/index"
+import { ListingEventType } from "@bloom-housing/backend-core/types"
+
+export const getLotteryEvent = (listing: FormListing) => {
+  let lotteryEvent = null
+  listing?.events.forEach((event) => {
+    if (event.type === ListingEventType.publicLottery) {
+      lotteryEvent = event
+    }
+  })
+  return lotteryEvent
+}
```

Comment:
There's a general helpers under `sites/partners/lib` where this can go. You could also update the syntax to be something like
```
export const getLotteryEvent = (listing: FormListing): ListingEvent | undefined => {
  const [lotteryEvent] = listing?.events.filter((event) => event.type === ListingEventType.publicLottery)
  return lotteryEvent
}
```